src/man/sssd-ldap-attributes.5.xml

```diff
@@ -636,6 +636,18 @@
                         </para>
                     </listitem>
                 </varlistentry>
+                <varlistentry condition="build_passkey">
+                    <term>ldap_user_passkey (string)</term>
+                    <listitem>
+                        <para>
+                            Name of the LDAP attribute containing the passkey
+                            mapping data of the user.
+                        </para>
+                        <para>
+                            Default: passkey
```

Comment:
Should it mention default value for LDAP/IPA/AD?

Like:
```
Default: homeDirectory (LDAP and IPA), unixHomeDirectory (AD)
```

Reply:
Fixed, thanks.